include/dlaf/permutations/general/impl.h

```diff
@@ -15,16 +15,21 @@
 
 #include "dlaf/blas/tile.h"
 #include "dlaf/common/index2d.h"
+#include "dlaf/communication/communicator.h"
+#include "dlaf/communication/message.h"
+#include "dlaf/communication/rdma.h"
```

Comment:
I removed `invert.cu` altogether as it's not currently needed and it will likely not be needed in it's current form in this kernel. https://github.com/eth-cscs/DLA-Future/pull/676/commits/3b2fe083f01a796218f01ab6bcd7adb20053bde3